tests/cucumber/steps/steps.js

```diff
@@ -4751,6 +4751,24 @@ module.exports = function getSteps(options) {
     }
   );
 
+  Then(
+    'I allow {int} more budget on that simulate request.',
+    async function (budget) {
+      this.simulateRequest.extraOpcodeBudget = budget;
+    }
+  );
+
+  Then(
+    'I check the simulation result has power packs extra-opcode-budget with extra budget {int}.',
+    async function (budget) {
+      assert.notDeepStrictEqual(undefined, this.simulateResponse.evalOverrides);
```

Comment:
nit: could do `assert.ok(this.simulateResponse.evalOverrides)` -- in my opinion this is a bit more clear, but essentially the same as what you already have